drink/src/session.rs

```diff
@@ -266,6 +223,9 @@ impl<R: Runtime> Session<R> {
             Ok(exec_result) => {
                 let address = exec_result.account_id.clone();
                 self.deploy_returns.push(address.clone());
+
+                self.transcoders.register(address.clone(), transcoder);
```

Comment:
nitpick: transcoders can be registered per code-hash. Not necessary to do that rn.

Reply:
but when calling a contract, I don't know its code-hash right away, do I? it would require to query the storage, right?

Reply:
I think when you deploy you could register the code hash.

Reply:
But this matters only for cases when:
1. You upload code.
2. Call `new` multiple times

Reply:
I know, but the problem is when you call a contract and you have to get the transcoder for its code-hash knowing only its address

Reply:
It would be a problem if you didn't deploy/instantiate that contract yourself but I don't think that's the case with drink? All contracts/addresses one will be interacting with are going to be created via this _session_ object, right? So when calling `deploy` or `instantiate` we can record the address and associate it with the code hash. You will need a separate map `Mapping<AccountId, Contract>`.